src/components/sbb-menu/sbb-menu.stories.js

```diff
@@ -0,0 +1,227 @@
+    </sbb-menu-action>
+    <sbb-menu-action icon="tickets-class-small" disabled={args.disabled}>
+      Tickets
+    </sbb-menu-action>
+    <sbb-menu-action icon="shopping-cart-small" amount={args.amount}>
+      Cart
+    </sbb-menu-action>
+    <sbb-divider />
+    <sbb-menu-action icon="exit-small">Log Out</sbb-menu-action>
+  </sbb-menu>,
+];
+
+const LongContentTemplate = (args) => [
+  triggerButton('menu-trigger-3'),
+  <sbb-menu trigger="menu-trigger-3" disable-animation={isChromatic()}>
+    <sbb-menu-action
+      icon={args.icon}
+      event-id="English"
+      disabled={args.disabled}
+      amount={args.amount}
+    >
+      English
+    </sbb-menu-action>
+    <sbb-menu-action>Deutsch</sbb-menu-action>
+    <sbb-menu-action>Français</sbb-menu-action>
+    <sbb-menu-action>Italiano</sbb-menu-action>
+    <sbb-menu-action>Rumantsch</sbb-menu-action>
+    <sbb-menu-action>Español</sbb-menu-action>
+    <sbb-menu-action>Português</sbb-menu-action>
+    <sbb-menu-action>日本語</sbb-menu-action>
+    <sbb-menu-action>한국어</sbb-menu-action>
+    <sbb-menu-action>广州话</sbb-menu-action>
+    <sbb-menu-action>Afrikaans</sbb-menu-action>
+    <sbb-menu-action>Svenska</sbb-menu-action>
+    <sbb-menu-action>Dansk</sbb-menu-action>
+    <sbb-menu-action>Nederlands</sbb-menu-action>
+    <sbb-menu-action>Suomi</sbb-menu-action>
+    <sbb-menu-action>українська мова</sbb-menu-action>
+    <sbb-menu-action>አማርኛ</sbb-menu-action>
+    <sbb-menu-action>ქართული ენა</sbb-menu-action>
+    <sbb-menu-action>Afrikaans</sbb-menu-action>
+    <sbb-menu-action>Svenska</sbb-menu-action>
+    <sbb-menu-action>Dansk</sbb-menu-action>
+    <sbb-menu-action>Nederlands</sbb-menu-action>
+    <sbb-menu-action>Suomi</sbb-menu-action>
+    <sbb-divider />
+    <sbb-menu-action icon="cross-small">Cancel</sbb-menu-action>
+  </sbb-menu>,
+];
+
+const EllipsisTemplate = (args) => [
+  triggerButton('menu-trigger-4'),
+  <sbb-menu trigger="menu-trigger-4" disable-animation={isChromatic()}>
+    <div style={userNameStyle}>Christina Müller</div>
+    <span style={userInfoStyle}>UIS9057</span>
+    <sbb-link href="https://www.sbb.ch/en" negative text-size="xs" variant="block">
+      Profile
+    </sbb-link>
+    <sbb-divider />
+    <sbb-menu-action icon={args.icon} href="https://www.sbb.ch/en">
+      View
+    </sbb-menu-action>
+    <sbb-menu-action icon="pen-small" disabled={args.disabled}>
+      Edit
+    </sbb-menu-action>
+    <sbb-menu-action icon="swisspass-small" amount={args.amount}>
+      Very long label that exceeds the maximum width of the menu, very long label that exceeds the
+      maximum width of the menu, very long label that exceeds the maximum width of the menu
+    </sbb-menu-action>
+    <sbb-divider />
+    <sbb-menu-action icon="cross-small">Cancel</sbb-menu-action>
+  </sbb-menu>,
+];
+
+export const Default = DefaultTemplate.bind({});
+Default.argTypes = defaultArgTypes;
+Default.args = { ...defaultArgs };
+Default.args.disabled = true;
```

Comment:
You can use spread operator to override values (same for CustomContent & LongContent).